pom.xml

```diff
@@ -409,13 +410,22 @@
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maves-surefire-plugin</artifactId>
+                    <artifactId>maven-surefire-plugin</artifactId>
                     <version>${maven-surefire-plugin-version}</version>
+                    <configuration>
+                        <includes>
+                            <include>**/*Test.java</include>
+                        </includes>
+                        <excludes>
+                            <exclude>**/*IT.java</exclude>
+                            <exclude>**/*IntegrationTest.java</exclude>
+                        </excludes>
+                    </configuration>
```

Comment:
We don't really need this. Surefire [already does it by default](https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html).

Reply:
On more note is that any configuration to the surefire configuration needs to be done on the parent's [pom.xml file](https://github.com/apache/camel/blob/main/parent/pom.xml). If needed, it's better to keep it in a single place.

Reply:
Surefire doesn't automatically exclude the pattern `*IntegrationTest.java`. I added all of the combinations just for completeness.

Reply:
This change, if necessary, should be done on the parent pom.